### ISSUE TITLE
KMNIST kNN 

### ISSUE THREAD

**Issue by dzisandy** [rois-codh/kmnist#10]
Adding PCA dimension reduction produces better Test Accuracy 
`x_train = x_train.reshape(-1, 784)`
`x_test = x_test.reshape(-1, 784)`
After this code rows:
`pca = PCA(n_components= 60, random_state= 0 )`
`x_train = pca.fit_transform(x_train)`
`x_test = pca.transform(x_test)`
The resulting is `Test accuracy: 0.9398`

**Comment by hardmaru** [comment]
Thanks, that’s a good datapoint to know!

**Comment by mxbi** [comment]
Added, it would be good to also have data points for MNIST and K-49

**Comment by dzisandy** [comment]
@mxbi Thanks for credit!

**Comment by mxbi** [comment]
See also #13 for K-49 and MNIST code/details
